_includes/main_scripts.html

```diff
@@ -64,7 +64,7 @@
 {%- endif -%}
 
 {%- if page.layout == 'page' or page.layout == 'our-team' or page.layout == 'our-journey' or page.layout == 'post' -%}
-<script src="https://printjs-4de6.kxcdn.com/print.min.js" crossorigin="anonymous"></script>
+<script src="https://printjs-4de6.kxcdn.com/print.min.js" integrity="sha384-7KBAAc532qVa9KK+IJzrJFWPLznPFKPl0vCBPrL1oOE7UKArUYXICoXh0qKdGBda" crossorigin="anonymous"></script>
```

Comment:
Related to my previous comment on version, can I confirm that the url
https://printjs-4de6.kxcdn.com/print.min.js
points to a deterministic asset? I.e. is the token `-4de6` a version identifier?